CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Change Log
 
+## v0.8.4 2020-12-08
+
+This release includes a number of improvements and bug fixes
+
+[full changelog](https://github.com/executablebooks/jupyter-book/compare/v0.8.3...6c30f554d86fe7d1a0e4ad05012a5de4133117d0)
+
+### New
```

Comment:
Particularly, for the new features, this should not just be a list of commits.
Imagine someone reading this, who is already using jupyter book; you want to give them a brief explanation of the new features, why they might want to use them, and add links to the documentation for further reading. 

Note for the document links, myst-parser is set up to "understand" relative markdown links, e.g. `[a link](docs/page.md)`

Reply:
we should document all of this in the contributing guidelines 👍 or for now, we can always just suggest "use a pattern that is similar to the latest releases"

Reply:
roger that -- we should update: https://executablebooks.org/en/latest/contributing.html#releases-and-change-logs as I read that as we wanted a list of commits for each category.

Reply:
> Note for the document links, myst-parser is set up to "understand" relative markdown links, e.g. [a link](docs/page.md)

@chrisjsewell do these links only reference a file. I am trying to use `tags` as links but doesn't seem to work (i.e. `(pdf:latex:config)=`)

Reply:
I'd say don't use tags, because they will not work when reading the file directly on GitHub.
For example, if you look at all the links in https://github.com/executablebooks/jupyter-book/blob/master/CHANGELOG.md#v080-2020-09-01, you can click on them and they will take you to the right page on github.

Reply:
there is actually the option in myst-parser to activate header anchors: https://myst-parser.readthedocs.io/en/latest/using/syntax-optional.html#auto-generated-header-anchors, but this is not "turned on" in jupyter book